metadata/service/server/init.yml

```diff
@@ -4,8 +4,8 @@ classes:
 - service.nagios.support
 parameters:
   _param:
-    nagios_password: r00tme
     nagios_username: nagiosadmin
+    nagios_password: r00tme
```

Comment:
You should not define any default password in service metadata to force user define it (or fail on UndefinedVariable error) on it's own on system level.